srcpkgs/make-ca/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'make-ca'
+pkgname=make-ca
+version=1.5
+revision=1
+wrksrc="make-ca-${version}"
+build_style=gnu-makefile
+make_install_args="SBINDIR=/usr/bin"
+hostmakedepends="help2man"
```

Comment:
`depends="p11-kit"`
Otherwise it errors out when cross compiling openjdk11
```
/usr/bin/install: creating directory '/builddir/jdk11u-jdk-11.0.4+11/ca/etc/ssl'
/builddir/jdk11u-jdk-11.0.4+11
Extracting OpenSSL certificates to /builddir/jdk11u-jdk-11.0.4+11/ca/etc/ssl/certs.../usr/bin/make-ca: line 826: /usr/bin/trust: No such file or directory
Failed!!!
Extracting GNUTLS server auth certificates to /builddir/jdk11u-jdk-11.0.4+11/ca/etc/pki/tls/certs/ca-bundle.crt.../usr/bin/make-ca: line 830: /usr/bin/trust: No such file or directory
Failed!!!
Extracting GNUTLS S-Mime certificates to /builddir/jdk11u-jdk-11.0.4+11/ca/etc/pki/tls/certs/email-ca-bundle.crt.../usr/bin/make-ca: line 834: /usr/bin/trust: No such file or directory
Failed!!!
Extracting GNUTLS code signing certificates to /builddir/jdk11u-jdk-11.0.4+11/ca/etc/pki/tls/certs/objsign-ca-bundle.crt.../usr/bin/make-ca: line 838: /usr/bin/trust: No such file or directory
Failed!!!
Extracting Java cacerts (JKS) to /builddir/jdk11u-jdk-11.0.4+11/ca/etc/pki/tls/java/cacerts.../usr/bin/make-ca: line 842: /usr/bin/trust: No such file or directory
Failed!!!
mv: cannot stat './ca/etc/pki/tls/java/cacerts': No such file or directory
```

Reply:
Also add `archs=noarch`